src/libs/actions/Task.js

```diff
@@ -519,10 +519,19 @@ function editTaskAssigneeAndNavigate(report, ownerAccountID, assigneeEmail, assi
                 reportName,
                 managerID: assigneeAccountID || report.managerID,
                 managerEmail: assigneeEmail || report.managerEmail,
+                pendingFields: {
+                    ...(assigneeAccountID && {managerID: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE}),
```

Comment:
@DylanDylann We are setting the PendingField based on `assigneeAccountID` status in the optimisticData but the same is not followed in the saveData.

We should be consistent here like this https://github.com/Expensify/App/blob/c435734d8cf360e964e20eea1fb54c541f6fa7ea/src/libs/actions/Task.js#L447

Reply:
Thanks for your comment, I updated

Reply:
@DylanDylann the above change should be in saveData pendingFields, not optimisticData pending fields

Reply:
@abdulrahuman5196 Sorry I am updating again. Sorry about my mistake

Reply:
`saveData` mean `successData`, right?

Reply:
Yeah. Sorry Typo

Reply:
NAB. Just to note. Kindly resolve the threads once fixed to avoid unwanted noises in the PR.